util/image-upload.js

```diff
@@ -1,9 +1,20 @@
 import { By, until } from "selenium-webdriver";
 import { defaultWait } from "../config.js";
 import path from "path";
+import { ExtendedBy } from "./selection.js";
 
-export async function uploadImage(driver, element_id, imagePath) {
-    const imageField = await driver.findElement(By.id(element_id));
+/**
+ * @param {any} driver 
+ * @param {string} element_id 
+ * @param {number} nthElement 
+ * @param {string} imagePath 
+ */
```

Comment:
What is the purpose of this documentation?

Reply:
Whoops I forgot to add the actual text.
I mostly added it for type checking without the need for a compiler though